src/User/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { List, ListItem } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+export const User = () => {
+  const [user, setUser] = React.useState([]);
+  React.useEffect(() => {
+    fetchUser();
+  }, []);
+  const fetchUser = async () => {
+    const response = await fetch(
+      `https://merey-todo-list.herokuapp.com/api/users`
```

Comment:
when you get your `id`, then you should provide it to the api:
`https://merey-todo-list.herokuapp.com/api/users/${id}`